docs/Roadmap.md

```diff
@@ -56,8 +56,14 @@ Extensions are not yet supported. Documents with extensions will get stored and
 ## Profiling
 Profiling is not supported yet, but the project aims to support storing profiles and validating against stored profiles.
 
+## Batch/Transaction
+currently there is no support for batch/transaction (https://www.hl7.org/fhir/http.html#transaction). This is something we are investigating and expect to support in future releases.
+
+## Conditional updates
```

Comment:
What about other conditional operations?

Reply:
+1.  Let's generalize this to all conditional operations (update, delete, and create).  They may have different priorities, but if we implement one, we'll likely implement all in the future.

Reply:
done. Please review and approve PR